domains/badboy.json

```
@@ -1,9 +1,10 @@
 {
   "description": "Bad Boy's Website",
-  "repo": "https://github.com/Bad-Boy-Codes/bad-boy-codes.github.io",
   "owner": {
     "username": "bad-boy-codes",
-    "email": "sonupanda078@gmail.com"
+    "email": "sonupanda078@gmail.com",
+    "discord": "445073800850046977",
+    "twitter": "BadBoyGoesBrr"
```

```suggestion
    "twitter": "BadBoyGoesBrr",
    "note": "My Discord ID is 445073800850046977."
```
optional, if you want to add ur id here then commit this